covquestions-js/yarn.lock

```diff
@@ -0,0 +1,13 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does npm install tolerates yarn.lock files?

Reply:
Yes, it simply ignores them.